front/src/components/MessagesBox.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { RootStateType } from '../redux/store';
+import Message from './Message';
+import styles from './MessagesBox.module.scss';
+
+const MessageBox = () => {
+  const messages = useSelector((state: RootStateType) => state.messages)
+
+  if (!messages?.length) {
+    return null;
+  }
+
+  return (
+    <div className={`${styles.messagesBox} container`}>
```

Comment:
talking about responsibility. Currently both MessageBox and Message component know about Redux. There is nothing bad in that, but I'd rather make `Message` taking `onClose` callback prop, so `MessageBox` could handle this, and `Message` would be simple "I show what I been told to"